src/cascadia/TerminalSettingsModel/KeyMapping.cpp

```diff
@@ -141,4 +141,40 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
 
         return keyModifiers;
     }
+
+    // Method Description:
+    // - Build a map of all the globalSummon actions.
+    // - quakeMode actions are included in this, but expanded to the equivalent
+    //   set of GlobalSummonArgs
+    // Arguments:
+    // - <none>
+    // Return Value:
+    // - a map of KeyChord -> ActionAndArgs containing all globally bindable actions.
+    Windows::Foundation::Collections::IMap<Control::KeyChord, Model::ActionAndArgs> KeyMapping::FetchGlobalHotkeys()
+    {
+        std::unordered_map<Control::KeyChord, Model::ActionAndArgs, KeyChordHash, KeyChordEquality> justGlobals;
+
+        for (const auto& [k, v] : _keyShortcuts)
+        {
+            if (v.Action() == ShortcutAction::GlobalSummon)
+            {
+                justGlobals[k] = v;
+            }
+            else if (v.Action() == ShortcutAction::QuakeMode)
+            {
+                // Manually replace the QuakeMode action with a globalSummon
+                // that has the appropriate action args.
+                auto args = winrt::make_self<GlobalSummonArgs>();
+
+                // We want to summon the window with the name "_quake" specifically.
+                args->Name(L"_quake");
+
+                Model::ActionAndArgs actionAndArgs{ ShortcutAction::GlobalSummon, *args };
+                justGlobals[k] = actionAndArgs;
+            }

```

Comment:
note to self, _ideally_ we'll populate `justGlobals` when we're populating the `ActionMap`. Which means that I'll just need to move this code over to `AddAction`.